src/core-unit/Unit/Unit/Unit.Comparison.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace System;
+
+partial struct Unit
+{
+    public int CompareTo(Unit other) => 0;
```

Comment:
Here and next. Shell we use "default" instead constant zero?

Reply:
Done: default is used instead of literal zero.